packages/react-contentkit/src/ElementWebframe.tsx

```diff
@@ -61,7 +62,7 @@ export function ElementWebframe(props: ContentKitClientElementProps<ContentKitWe
 
             // For security reasons, only iframe from our integrations domains are allowed
             // to send and receive messages
-            if (!renderer.security.firstPartyDomains.includes(origin.host) && 0) {
+            if (!renderer.security.firstPartyDomains.includes(origin.host)) {
```

Comment:
Glad you removed that `&& 0` because I'm not sure what that was doing.

Reply:
It looks like it's my mistake, probably while developing and forgot to remove it 😬